metashare/api/models.py

```diff
@@ -655,8 +656,16 @@ def remove_scratch_org(self, error):
     def queue_refresh_org(self):
         from .jobs import refresh_scratch_org_job
 
+        self.currently_refreshing_org = True
+        self.save()
+        self.notify_changed()
         refresh_scratch_org_job.delay(self)
 
+    def finalize_refresh_org(self):
+        self.currently_refreshing_org = False
+        self.save()
+        self.notify_changed()
+
```

Comment:
@wlonk I'm getting the org to update successfully, but curious if this should this be broadcasting a websocket event to listen to on the frontend?

Reply:
It should; the logic is in the `PushMixin` class. It should send a `SCRATCH_ORG_UPDATE` event, with the new serialization, including `currently_refreshing_org` as `False`.

Reply:
OK, I thought so.  I reached my org limit so have to start over :frowning_face: I'll try again and see if the event comes through.  Though it didn't come through when there was an error, and maybe it should have?